packaging/installer/kickstart.sh

```diff
@@ -525,6 +526,27 @@ update() {
   fi
 }
 
+uninstall() {
+  get_system_info
+  detect_existing_install
+
+  if [ $INTERACTIVE = 0 ]; then
+    FLAGS="--yes --force"
+  else
+    FLAGS="--yes"
+  fi
+
+  if [ -f "${INSTALL_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh" ]; then
+    echo "Found existing netdata-uninstaller. Runing it.."
+    ${INSTALL_PREFIX}/usr/libexec/netdata/netdata-uninstaller.sh $FLAGS
+  else
+    echo "Downloading netdata-uninstaller ..."
+    wget https://raw.githubusercontent.com/netdata/netdata/master/packaging/installer/netdata-uninstaller.sh -O /tmp/netdata-uninstaller.sh
+    chmod +x /tmp/netdata-uninstaller.sh
+    /tmp/netdata-uninstaller.sh $FLAGS
```

Comment:
If possible this should use the temporary directory handling used throughout the rest of the code.